_posts/wiki/2020-03-23-robotic-skin.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Robotic Skin
+description: Research Page
+tags: [robotics,panda,simulator,research]
+permalink: robotic_skin.html
+---
+
+{% include image.html url="research/skin_unit.png" max-width="100%" description="Flexible skin unit created by etching the circuit diagram into a copper sheet and then encasing it in the flexible polymer ecoflex." %}
+
+
+Robots have been transitioning into human-populated environments and replacing their physical cages used to keep humans safe with complex perception and software. However, current solutions are computationally expensive, prone to occlusion, and require setup overhead. Robots require compact, self-contained sensing of nearby space to provide inherently save interactions. In this work, we present the first prototype of a flexible, conformant whole-body artificial skin for collaborative robotics. In order to locate all skin units on the surface of a robot, a novel kinematically calibration algorithm is required to reduce setup time.
+
+# Contributions
+
+1. <strong>Kinematic Calibration:</strong> We present a framework for automatic kinematic calibration that leverages an Inertial Measurement Unit (IMU) to automatically locate skin units, of arbitrary number, along the surface of a robot.
+
+1. <strong>Skin Unit:</strong> We present a new <strong>flexible artificial skin</strong> that can conform to multiple surfaces. The skin utilizes an IMU and a proximity sensor to sense obstacles at a distance. Future prototypes will embed additional capabilities (e.g. pressure sensing, temperature sensing, tactile sensing...)
+
+1. <strong>Control Example:</strong> We demonstrate an example control scenario where a robot arm utilizes artificial skin to <strong>avoid collision</strong>  with a human, increasing operational safety with and around people.
+
+
+# Kinematic Calibration
```

Comment:
It doesn't explain what it does. It'd be nice to have some equations

Reply:
@watakandai Are the error equations that we used in the poster session still accurate to use in this part of the research page? If not, what else do you think we should use in this section to express how calibration is done? 

![Screenshot from 2020-04-05 15-17-02](https://user-images.githubusercontent.com/37485219/78510329-adca6400-7751-11ea-8644-5b844de69ad4.png)

Reply:
I don't

Reply:
I strongly disagree to post equations and our current method online even though we haven't published our paper.
Recently, I heard from one of my friend that his method was stolen from his co-author (professor). We should not address any of our method here on the website.

Reply:
@alecive What do you think?

Reply:
Ok then, let's wait to add equations then. I would still like to have some technical content though--any thoughts on how to make this happen?